jupyterlab/tests/test_announcements.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import hashlib
+import json
+from unittest.mock import MagicMock, patch
+
+import tornado
+
+from . import Response, to_async_mock
+
+FAKE_ATOM_FEED = b"""<?xml version="1.0" encoding="utf-8"?><feed xmlns="http://www.w3.org/2005/Atom" ><generator uri="https://jekyllrb.com/" version="3.9.2">Jekyll</generator><link href="https://jupyterlab-contrib.github.io/assets/feed.xml" rel="self" type="application/atom+xml" /><link href="https://jupyterlab-contrib.github.io/assets/" rel="alternate" type="text/html" /><updated>2022-11-02T15:14:50+00:00</updated><id>https://jupyterlab-contrib.github.io/assets/feed.xml</id><title type="html">JupyterLab News</title><subtitle>Subscribe to get news about JupyterLab.</subtitle><entry><title type="html">Thanks for using JupyterLab</title><link href="https://jupyterlab-contrib.github.io/assets/posts/2022/11/02/demo.html" rel="alternate" type="text/html" title="Thanks for using JupyterLab" /><published>2022-11-02T14:00:00+00:00</published><updated>2022-11-02T14:00:00+00:00</updated><id>https://jupyterlab-contrib.github.io/assets/posts/2022/11/02/demo</id><content type="html" xml:base="https://jupyterlab-contrib.github.io/assets/posts/2022/11/02/demo.html">&lt;h1 id=&quot;welcome&quot;&gt;Welcome&lt;/h1&gt;
+
+&lt;p&gt;Thanks a lot for your interest in JupyterLab.&lt;/p&gt;</content><author><name></name></author><category term="posts" /><summary type="html">Big thanks to you, beloved JupyterLab user.</summary></entry></feed>"""
+
+FAKE_JUPYTERLAB_PYPI_JSON = b"""{ "info": { "version": "1000.0.0" } }"""
+
+
+@patch("tornado.httpclient.AsyncHTTPClient")
+async def test_NewsHandler_get_success(mock_client, labserverapp, jp_fetch):
+
+    mock_client.return_value = MagicMock(
+        spec=tornado.httpclient.AsyncHTTPClient,
+        fetch=to_async_mock(Response(FAKE_ATOM_FEED)),
+    )
+
+    response = await jp_fetch("lab", "api", "news", method="GET")
+
+    assert response.code == 200
+    payload = json.loads(response.body)
+    assert payload["news"] == [
+        {
+            "createdAt": 1667397600000.0,
+            "message": 'Thanks for using JupyterLab\n\nBig thanks to you, beloved JupyterLab user.  \nSee <a href="https://jupyterlab-contrib.github.io/assets/posts/2022/11/02/demo.html" target="_blank" rel="noreferrer">full post</a> for more details.',
+            "modifiedAt": 1667397600000.0,
+            "type": "info",
+            "options": {
+                "data": {
+                    "id": "https://jupyterlab-contrib.github.io/assets/posts/2022/11/02/demo",
```

Comment:
Should this still be `jupyterlab-contrib` or `jupyterlab` now?

```suggestion
                    "id": "https://jupyterlab.github.io/assets/posts/2022/11/02/demo",
```

![image](https://user-images.githubusercontent.com/591645/202401343-21b1ff2b-b151-468b-95c9-7ab93c8ab95d.png)

Reply:
This is using the mock `FAKE_ATOM_FEED` for testing - there are no requests done actually. So it does not matter.

Reply:
Also this 404 was on Binder after clicking on the link. But I guess this is because the current feed still contains links to `jupyterlab-contrib` for now: https://jupyterlab.github.io/assets/feed.xml

![image](https://user-images.githubusercontent.com/591645/202469107-ded2d5b0-2abe-4cef-b584-ce4ba06944f9.png)

Reply:
I updated the asset project this morning